posthog/hogql/test/_test_parser.py

```diff
@@ -1283,6 +1283,16 @@ def test_order_by(self):
                     end=14,
                 ),
             )
+            # Note that the parser will skip anything after `--`, so the `DESC` behind will not be parsed
```

Comment:
I added a test case here because I thought this might help people understand the parser slightly better, let me know if yall would prefer this to be removed